tests/add.bats

```diff
@@ -58,6 +60,22 @@ load helpers
   run_buildah rm $newcid
 }
 
+@test "add-multiple-files to destination not ending with slash" {
+  createrandom ${TEST_SCRATCH_DIR}/randomfile
+  createrandom ${TEST_SCRATCH_DIR}/other-randomfile
+  createrandom ${TEST_SCRATCH_DIR}/third-randomfile
+
+  run_buildah from $WITH_POLICY_JSON scratch
+  cid=$output
+  run_buildah mount $cid
+  root=$output
+  run_buildah config --workingdir / $cid
+  run_buildah 125 add $cid ${TEST_SCRATCH_DIR}/* /test
```

Comment:
Without quoting, the globbing here is done by the shell, not by buildah, so buildah doesn't see the globbing pattern.